packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1349,11 +1365,11 @@ export class ContainerRuntime
 			}
 		}
 
-		this.deltaManager.on("readonly", (readonly: boolean) => {
+		this._deltaManager.on("readonly", (readonly: boolean) => {
```

Comment:
Same question as above - should this be using proxy and never send ops? I.e. what is the boundary of lies?